docs/troubleshooting.rst

```diff
@@ -14,7 +14,6 @@ We have a script that attempts to fix some of the most common problems with
 Kuma that are described below::
 
   pushd .
-  # open https://www.youtube.com/watch?v=ckIMuvumYrg
```

Comment:
I'm sorry--I should have added comments to the PR.

I was bummed about this, but the video at that location has been removed. I thought about replacing it with something else, but generally I think we've got too much documentation and it's probably worth while removing things.

Anyhow, I'm game for replacing it with something else. Maybe this?:

https://www.youtube.com/watch?v=DtRNg5uSKQ0